kahuna/public/js/components/gr-confirmation-modal/gr-confirmation-modal.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from "react";
+import * as angular from "angular";
+import { react2angular } from "react2angular";
+import "./gr-confirmation-modal.css";
+import {useEffect, useState, useRef} from "react";
```

Comment:
You can consolidate the two react imports by adding FC to the imports in line 5 and changing line 16 to match